lib/VCMIDirs.cpp

```diff
@@ -15,6 +15,8 @@
 #include "iOS_utils.h"
 #endif
 
+#include <mutex>
```

Comment:
Move to global.h perhaps?

Reply:
grep shows that this include isn't used anywhere else. Should it still be moved?

Reply:
For standard library includes - yeah, I think so